treeherder/perf/models.py

```diff
@@ -156,11 +201,13 @@ class PerformanceDatum(models.Model):
     class Meta:
         db_table = 'performance_datum'
         index_together = [
-            # this should speed up the typical "get a range of performance datums" query
+            # Speeds up the typical "get a range of performance datums" query
             ('repository', 'signature', 'push_timestamp'),
-            # this should speed up the compare view in treeherder (we only index on
+            # Speeds up the compare view in treeherder (we only index on
             # repository because we currently filter on it in the query)
-            ('repository', 'signature', 'push')]
+            ('repository', 'signature', 'push'),
+            # Speeds up data cycling
+            ('push_timestamp',)]
```

Comment:
I think this should be enabled on the `push_timestamp` field itself (`db_index=True`) instead of being done here.

Reply:
Got it.